SPECS/libkcapi/libkcapi.spec

```diff
@@ -209,6 +211,11 @@ rm -f                            \
   %{buildroot}%{_bindir}/sha*sum      \
   %{buildroot}%{_bindir}/sm*sum
 
+# Create hard-links to alias dracut-expected sha* bins to new kcapi-hasher bin.
+for app in %{apps_hmaccalc}; do
+  ln %{buildroot}%{_bindir}/kcapi-hasher %{buildroot}%{_bindir}/$app || exit 1;
```

Comment:
Should this be `ln -s` to create symbolic links instead of hard links?

Reply:
Soft links don't work with the dracut fips logic. It'll follow the soft link to the real binary, kcapi-hasher, and then try to find kcapi-hasher.hmac.

![image](https://github.com/microsoft/azurelinux/assets/37827103/a598540e-42b4-4785-9f21-b4ad3b03bfdb)